docs/rules/prefer-number-properties.md

```diff
@@ -93,6 +93,13 @@ Default: `false`
 
 Pass `checkInfinity: true` to enable check on `Infinity`.
 
+### checkNaN
+
+Type: `boolean`\
+Default: `false`
+
+Pass `checkNaN: true` to enable check on `NaN`.
+
 #### Fail
```

Comment:
Your docs change is in the wrong place, the `Fail` section below is for `checkInfinity`, so you should at it below and add your own `Fail` examples.